ios/RNRadar.m

```diff
@@ -81,6 +81,10 @@ - (void)didLogMessage:(NSString *)message {
     }
 }
 
+RCT_EXPORT_METHOD(setLogLevel:(NSInteger *)level) {
+    [Radar setLogLevel:level];
```

Comment:
@kochis May want to cast this to `(RadarLogLevel)` (although may not strictly be necessary)